app/views/set_ups/show.html.erb

```diff
@@ -1,2 +1,39 @@
 <h1>SetUps#show</h1>
 <p>Find me in app/views/set_ups/show.html.erb</p>
+<h1>Time Frame</h1>
+<%= @set_up.start_date %>
+<%= @set_up.end_date %>
+<h1>Clinic</h1>
+<% @set_up.clinics.each do |clinic| %>
+  <%= clinic.name %>
+  <br>
+<% end %>
+<h1>Completion Rate</h1>
+<%= ((@set_up.health_checks.count) / (@set_up.company.users.count).to_f * 100).ceil %>
+<br>
+<table class="mt-4">
+  <tr>
+    <th>Employee Name</th>
+    <th>Health Check Date</th>
+    <th>Status</th>
+  </tr>
+  <% @set_up.company.users.each do |user| %>
+    <tr>
+      <td class="employeename">
```

Comment:
Is the style (font, color, spacing, etc.) of the employee name different from the style of the health check date? If they are the same, I think it is fine to have the same class, like table-content-text.